handlers/oess/src/main/java/orca/handlers/network/oess/tasks/UrnRemapperTask.java

```diff
@@ -0,0 +1,89 @@
+package orca.handlers.network.oess.tasks;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+import orca.shirako.plugins.config.OrcaAntTask;
+
+import org.apache.tools.ant.BuildException;
+
+/**
+ * Remap URNs based on a property file
+ * For now the property file has one property with an even number
+ * of comma-separated URNs. The odd-ones are 'from', the even ones are 'to'
+ * @author ibaldin
+ *
+ */
+public class UrnRemapperTask extends OrcaAntTask {
```

Comment:
Can we use the Oscars [UrnRemapperTask](https://github.com/RENCI-NRIG/orca5/blob/orca-5.1.6/handlers/oscars/src/main/java/orca/handlers/network/oscars/tasks/UrnRemapperTask.java) without duplicating the code?